src/ssl.c

```diff
@@ -36287,7 +36418,11 @@ void wolfSSL_PKCS7_free(PKCS7* pkcs7)
         XFREE(p7, NULL, DYNAMIC_TYPE_PKCS7);
     }
 }
-
+void wolfSSL_PKCS7_SIGNED_free(PKCS7_SIGNED* p7)
+{
+    wolfSSL_PKCS7_free((PKCS7*)p7);
```

Comment:
PKCS7 and PKCS7_SIGNED are the same types?

Reply:
Yes, they are the same types.